src/components/organisms/UiList/_internal/UiListItemSuffix.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <UiButton class="ui-button--text ui-button--small ui-list-item-suffix">
+    <!-- @slot Use this slot to replace label template. -->
+    <slot
+      name="label"
+      v-bind="{ label }"
+    >
+      {{ label }}
```

Comment:
```suggestion
      <span v-bind="labelAttrs">{{ label }}</span>
```
I find 1 use case when we need that version. When you have a button without label you should add it
* as `aria-label` to button 
* add `visual-hidden` to label, and that, why we need this span. WDYT?

Reply:
nice, thanks 😀